app/views/layouts/modal_create.js.erb

```diff
@@ -1,9 +1,22 @@
-$('.popup_content_visible').popup('hide');
-$('body').prepend(
+$('.contact_heading').html(
   "<%=
-    j render 'layouts/partials/flash',
-             type: 'success',
+    j render 'layouts/partials/h2',
+             type: 't.contacts.create.success',
              content: yield
   %>"
 );
+$('.js-report-overlay_close').css("background-color","#8cbd86");
+$('.JS-Single-Click-Button__active').css("background-color","#8cbd86");
```

Comment:
Etwas viel statisches CSS. Lass uns da einfach ne Klasse verteilen und das CSS in den entsprechenden Dateien lassen. Ums styling musst du dich auch nicht wirklich kümmern. Im Zweifelsfall einfach an Marcus weiterleiten.